tools/Utilities/HostsUtility/Views/HostsFileEditorSettingsWindow.xaml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<windowex:WindowEx
+    x:Class="DevHome.HostsFileEditor.Views.HostsFileEditorSettingsWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.HostsFileEditor.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+    xmlns:ui="using:CommunityToolkit.WinUI"
+    xmlns:common="using:DevHome.Common.Windows"
+    xmlns:windowex="using:WinUIEx"
+    MinWidth="400"
+    MinHeight="480"
+    mc:Ignorable="d">
+    <Window.SystemBackdrop>
+        <MicaBackdrop />
+    </Window.SystemBackdrop>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <common:WindowTitleBar
+            x:Name="AppTitleBar"
+            Title="{x:Bind WindowTitle}"
+            IconSource="ms-appx:///Assets/HostsUILib/Hosts.ico"
+            Grid.Row="0" />
+
+        <StackPanel
+            Padding="60,0"
+            Grid.Row="1">
+            <TextBlock
+                Padding="6,8"
+                Style="{ThemeResource SubtitleTextBlockStyle}"
+                Text="Hosts Utility Settings"
+                MaxWidth="{ThemeResource MaxPageContentWidth}"
+                Margin="{ThemeResource ContentPageMargin}"/>
+                <ScrollViewer VerticalScrollBarVisibility="Auto" HorizontalScrollBarVisibility="Auto" >

```

Comment:
When we go over this, can we look at the UI here? I want to be sure the scroll bar is where it should be.